### PR TITLE
Add code of conduct

### PR DESCRIPTION
**Pull request by avahoffman**
Added a code of conduct, based on Bioconductor's CoC

### CODE REVIEW

File: resources/code_of_conduct.Rmd [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+## Ombudsperson
+
+We understand that you may want a person unrelated to DaSEH overseeing your conflict resolution. To facilitate this, we encourage community members to reach out to our designated third party service (EthicsPoint) contracted by Fred Hutch. See more information [here](https://secure.ethicspoint.com/domain/media/en/gui/89211/index.html). They can assist community members by offering consultation, mediation, coaching and referrals to other resources.
+
+## Incident Procedure
+
+The organizers follow a standard procedure to deal with conflicts. This procedure is briefly summarized below. Incident procedures are based on the book “How to Respond to Code of Conduct Reports” by Valerie Aurora and Mary Gardiner.
+
+#### Report Acknowledgement
```
Comment (carriewright11) [inline resources/code_of_conduct.Rmd:93]: do we want to use the american spelling (Acknowledgment)? this word always trips me up :P

File: resources/dictionary.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+Acknowledgement
```
Comment (carriewright11) [inline resources/dictionary.txt:1]: Acknowledgment?

Comment (carriewright11) [reply resources/dictionary.txt:1]: just american vs british - either way is fine, just pointing it out :)

File: resources/code_of_conduct.Rmd [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+We expect everyone in the DaSEH community to comply with the Code of Conduct when participating in events and online communication platforms.
+
+## Reporting
+
+If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. You can make a report either anonymously or personally. All reports will be reviewed by the organizers and will be kept confidential.
+
+#### Electronically
+
+You can make an anonymous or non-anonymous report via the following link: https://forms.gle/WBymAnzfWYpfWk4b6. It is a free-form text box that will be forwarded to the organizers. If you are uncomfortable reporting to the organizers as a group, you can contact any individual instructor via email or a direct message on the Slack workspace (https://daseh.slack.com/). Please include screenshots/copies of all relevant electronic conversations whenever possible (you don’t need to compromise your anonymity!).
```
Comment (cansavvy) [inline resources/code_of_conduct.Rmd:71]: I really like this form. I feel like we could use this for the ITN slack. Do you mind sharing it with me? (putting it in the shared drive might work??)

File: resources/code_of_conduct.Rmd [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+We can’t follow up an anonymous report with you directly, but we will fully investigate it and take whatever action is necessary to prevent a recurrence.
+
+#### Personal Report (for any DaSEH events: in-person or virtual)
+
+You can make a personal report to any member of the instructors or organizers present at an event.
```
Comment (cansavvy) [inline resources/code_of_conduct.Rmd:77]: If an organizer is the problem who do they report to?

**Review by cansavvy — APPROVED** [review]
Looks great to me! Only one minor comment.

File: resources/code_of_conduct.Rmd (hunk 4 above)
Comment (avahoffman) [reply resources/code_of_conduct.Rmd:77]: There's Hutch's independent service mentioned below but I'll make that more clear!

File: resources/dictionary.txt (hunk 2 above)
Comment (avahoffman) [reply resources/dictionary.txt:1]: This one always trips me up, thanks!!

**Pull request merged**